src/utils/constants.ts

```diff
@@ -3,5 +3,9 @@ import { BigInt } from '@graphprotocol/graph-ts';
 export const bigZero = BigInt.fromI32(0);
 export const bigOne = BigInt.fromI32(1);
 export const AddressZero = '0x0000000000000000000000000000000000000000';
-export const TRADE_TYPE_BUY = 'Buy';
-export const TRADE_TYPE_SELL = 'Sell';
+export const ERC20AssetId = "-1"
```

Comment:
I believe this is not correct, we can't have negative numbers as the assetID that we get from the chain(it's uint256, so unsigned). Instead of using the assetID to check if something is an erc20 token, we'd need to check that the makerAsset address == collateralToken

Reply:
Great catch! I just pushed some changes to handle this without hardcoding the collateral address